redundant-bmc/src/manager.cpp

```diff
@@ -23,9 +26,25 @@ sdbusplus::async::task<> Manager::startup()
 
     redundancyInterface.role(determineRole());
 
+    spawnRoleHandler();
+
     co_return;
 }
 
+void Manager::spawnRoleHandler()
+{
+    if (redundancyInterface.role() == Role::Active)
+    {
+        handler = std::make_unique<ActiveRoleHandler>(ctx, services);
+    }
+    else
```

Comment:
I think it's wise to check if the role is passive or Should we start the passive BMC services in that default case?

Reply:
I guess to be safe I should check that.  It really shouldn't hit that though so maybe I will throw an exception so it crashes and restarts and we'll notice that something would need to be fixed.